src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -354,6 +365,141 @@ bool AsyncRPCOperation_sendmany::main_impl() {
     LogPrint("zrpcunsafe", "%s: private output: %s\n", getId(), FormatMoney(z_outputs_total));
     LogPrint("zrpc", "%s: fee: %s\n", getId(), FormatMoney(minersFee));
 
+
+    /**
+     * SCENARIO #0
```

Comment:
Nit: This method is already so ginormous. It would be nice if these scenarios were extracted in to methods. Especially new ones that we are adding.

Reply:
Falls under #2979.